src/pcapi/notifications/push/backends/batch.py

```diff
@@ -27,7 +33,27 @@ def __init__(self) -> None:
         self.headers = {"Content-Type": "application/json", "X-Authorization": settings.BATCH_SECRET_API_KEY}
 
     def update_user_attributes(self, user_id: int, attribute_values: dict) -> None:
-        def make_post_request(api: BatchAPI) -> None:
+        for api in (BatchAPI.ANDROID, BatchAPI.IOS):
+            url = f"{settings.BATCH_API_URL}/1.0/{api.value}/data/users/{user_id}"
+            try:
+                http_request = CloudTaskHttpRequest(
+                    http_method=tasks_v2.HttpMethod.POST,
+                    headers=self.headers,
+                    url=url,
+                    json={"overwrite": False, "values": attribute_values},
+                )
+                enqueue_task(BATCH_CUSTOM_DATA_QUEUE_NAME, http_request)
+
+            except Exception as exc:  # pylint: disable=broad-except
+                logger.error(
```

Comment:
```suggestion
                logger.exception(
```

... ainsi, on aura un traceback dans Sentry qui pourra nous être utile.